### DISCUSSION TITLE
Welcome to sd-webui-lora-block-weight Discussions!

### DISCUSSION THREAD

**Discussion by hako-mikan** [hako-mikan/sd-webui-lora-block-weight#123] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hyfevian** [comment]
![4e537dbf53e6f46abe4a917411a5a305](https://github.com/hako-mikan/sd-webui-lora-block-weight/assets/160686617/3e48dc16-1b47-43a2-8de3-046e25ba007b)
pls,i can fix it
